client/go/kserve-api/resnet_labels.txt

```diff
@@ -0,0 +1,1000 @@
+tench, Tinca tinca
```

Comment:
It's not good to add 1k duplicated lines again, same as with java. @michalkulakowski has prepared PR with de-duplication of label files, can you use similar approach?